types/applications/application.d.ts

```diff
@@ -381,7 +381,7 @@ declare namespace Application {
      * The default pixel height for the rendered HTML
      * @defaultValue `null`
      */
-    height: number | null;
+    height: string | number | null;
```

Comment:
What's the reason for this change?

Reply:
Ah, `SidebarDirectory` passes `'auto'`. Maybe it should be `number | 'auto' | null`?

Reply:
What if you pass a measurement with units? Like `"3rem"`? Is that valid?

Reply:
I only found explicit checks for "auto" so far. Using any other CSS units thus seems dangerous, since in some cases there's just a "px" appended to the string number. Also apparently it can be "auto" for width as well, but the only place where that's used or even check for is in `Draggable`.

Reply:
I'd go with `'auto'` then. If somebody wants to use something different, they can just create an issue.